src/main/proto/io/github/anki/anki/api/nats/v1/deck/event.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+
+package io.github.anki.anki.api.nats.v1.deck.event;
+
+option java_multiple_files = true;
+
+message DeckEvent {
+  string deck_id = 1;
+  DeckEventType deck_event_type = 2;
+}
+
+enum DeckEventType {
```

Comment:
start from 1 and 0 left for unspecified 
https://protobuf.dev/programming-guides/dos-donts/#unspecified-enum